README.md

```diff
@@ -7,3 +7,158 @@ This repo models a process allowing external guidance authors, approvers and pub
 ### License
 
 This code is open source software licensed under the [Apache 2.0 License]("http://www.apache.org/licenses/LICENSE-2.0.html").
+
+## Endpoints
+
+### `POST: /external-guidance/scratch`
+
+#### Purpose
+
+Allows Ocelot to send over a scratch process and check how it looks in the browser.
+The generated ID is returned to the caller.
+
+#### Request Body
+
+This is the json string representing the Ocelot process
+
+#### Success Response
+
+**HTTP Status**: `201`
+
+**Example Response Body**:
```

Comment:
Please can you add something around the `Location` header as this is what they need to redirect the user to so they can view what was pushed up.

Reply:
updated, will push shortly